app/Http/Controllers/Secretariat/UserController.php

```diff
@@ -171,6 +171,11 @@ public function updateEducationalInformation(Request $request, User $user): Redi
             'extra_information' => ['nullable', 'string', 'max:1500'],
         ]);
 
+        // whether Neptun code is unique
+        if (EducationalInformation::where('neptun', $request->neptun)->where('user_id', '<>', $user->id)->exists()) {
+            return redirect()->back()->with('message', 'A megadott Neptun-kód már létezik! Ha a kód az Öné, lépjen be a korábbi fiókjával.');
```

Comment:
->with('error')...
It will display a warning sign and the toast have a longer display period.